utils/audio.js

```diff
@@ -1,18 +1,101 @@
 'use strict';
 
-module.exports = {
-  buildSfxCollection(game, componentName, clipCount) {
-    // FIXME: Make this a class.
+class Audio {
```

Comment:
Possibly makes sense, though it would be better to build it more like a singleton in that case, so we don't have to keep initializing it in every state... That's probably most easily done by turning it back into an object and adding a `setGame` method that we call in `index.js` that sets `this.game`.